### ISSUE TITLE
Support a CI Platform (circle CI) (depends on #1)

### ISSUE THREAD

**Issue by jermnelson** [LD4P/bfe#5]
_From @jermnelson on August 2, 2018 17:26_

Need to support CI (continuous integration) platform like [CircleCI](https://circleci.com/) and [Travis](https://travis-ci.org/)

Needs #1  Needs #2

_Copied from original issue: sul-dlss/bfe#5_

**Comment by cmharlow** [comment]
If possible, I'd like to see us try to pick back up CirleCI for Sinopia overall, given our usage with it for AWS + docker (see [TACO's work with this](https://github.com/sul-dlss-labs/taco/blob/master/.circleci/config.yml)).

**Comment by ndushay** [comment]
yes, i assume the best CI platform depends on the context in which we will be working/deploying.
